README.md

```diff
@@ -78,6 +78,15 @@ Right click on your AKS cluster and click on **Configure CI/CD Workflow** to cre
 
 Right click on your AKS cluster and click on **Browse CI/CD Workflows** to view details of all existing Continous Integration / Continous Deployment (CI/CD) pipelines that are configured against your cluster.
 
+### Install Azure Service Operator
+
+Right click on your AKS cluster and click on **Install Azure Service Operator** to easily deploy the latest version of Azure Service Operator (ASO) on your AKS cluster and provision and connect applications to Azure resources within Kubernetes. Upon selecting the option, a web view will load requesting your input on Service Principal App Id and the password. Fill out these required fields and click **Submit** to kick off the installation of Azure Servic Operator.
```

Comment:
Rather than "requesting your input on..." I'd suggest "you'll be prompted for..."?
```suggestion
Right click on your AKS cluster and click on **Install Azure Service Operator** to easily deploy the latest version of Azure Service Operator (ASO) on your AKS cluster and provision and connect applications to Azure resources within Kubernetes. When you select this option, you'll be prompted for a service principal for ASO to use when performing Azure resource operations. This service principal must have appropriate permissions (typically Contributor at suitable scope). Fill out the service principal details and click **Submit** to kick off the installation of Azure Servic Operator.
```

Reply:
Thank you for the suggestion! I committed the change.